sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_paging.py

```diff
@@ -0,0 +1,17 @@
+from azure.core.paging import ItemPaged
+from azure.core.async_paging import AsyncItemPaged
+
+class AnalyzeHealthcareResult(ItemPaged):
```

Comment:
interesting paging, i think this can be clearer with coming autorest improvements, but if it's working rn I'm not going to touch anything

Reply:
Haha, it's probably a little messier than it should be but it works for now.  I was thinking it might be better if all of the paging callbacks could live in a class instead of just regular functions in _response_handlers.py, but I didn't want to invest too much time in that for this version.  I'm also wondering if the ItemPaged objects as results would be confusing for the user since the result is not consistent with the way the other APIs are returning classes from _models.py.